### DISCUSSION TITLE
Can "echo" be enabled to support Palo Alto Panorama Backup?

### DISCUSSION THREAD

**Discussion by Robert92374** [drakkan/sftpgo#1661] [Q&A]
I'm trying to schedule backups of my Panorama config and the only options are SCP and FTP. When testing, it fails using SCP, and the SFTPGo logs indicate the issue is with the echo command the panorama is trying to use. Logs are below.

According to [SFTPGo's documentation](https://gitee.com/OtherCopy/sftpgo/blob/2.0.x/docs/ssh-commands.md), echo can't be enabled in the config file (sftpgo.json). Is there any other way to enable this command to allow the Panorama to connect?

{
    "level": "debug",
    "time": "2024-06-16T06:46:10.080",
    "sender": "sshd_exec",
    "message": "new ssh command: \"echo\" args: [paloaltonetworks > /ssh-export-test.txt] num args: 3 user: panorama, error: <nil>"
}{
    "level": "info",
    "time": "2024-06-16T06:46:10.080",
    "sender": "sshd_exec",
    "message": "ssh command not enabled/supported: \"echo\""
}{
    "level": "error",
    "time": "2024-06-16T06:46:10.080",
    "sender": "sshd_exec",
    "message": "unable to unmarshal ssh command, close fs, err: <nil>"
}

**Comment by Macleykun** [comment]
hey, i know how to add this. is this still something you're looking for?

**Reply by Robert92374** [reply]
It turns out the test from the Panorama fails due to this error, but the actual backup succeeds. I'd still be interested in how you can fix it, though.
